standard/classes.md

```diff
@@ -874,10 +874,10 @@ All members of a generic class can use type parameters from any enclosing class,
 >     static void Main()
 >     {
 >         C<int> x1 = new C<int>(1);
->         Console.WriteLine(x1.f1);              // Prints 1
+>         System.Console.WriteLine(x1.f1);              // Prints 1
```

Comment:
I definitely want to hold off on this until we've had a discussion about it.

Reply:
removed qualifier and marked "Incomplete"